ansible-scylla-node/tasks/Debian.yml

```diff
@@ -74,15 +69,91 @@
     when: scylla_version == 'latest' and scylla_edition == 'enterprise'
 
   - name: Install specified OSS Scylla
-    apt:
-      name: "scylla={{ scylla_version }}*"
-      state: present
+    block:
+      - name: Get dependencies of scylla package
+        shell: apt depends scylla={{ scylla_version }}*
```

Comment:
should probably be `scylla={{ scylla_version }}-*` 
Otherwise if the version is 2022.1.1 you will get dependencies for 2022.1.1 and 2022.1.10, 2022.1.11, ... 2022.1.19 too.

Reply:
will fix

Reply:
look ups of version are now used